src/components/meetupSection.js

```diff
@@ -57,83 +59,95 @@ const MeetupSection = () => (
-        sx={{
-          display: "grid",
-          gridGap: "3rem 2rem",
-          gridTemplateColumns: [
-            "auto",
-            null,
-            `repeat(auto-fit, minmax(25em, 1fr))`,
-          ],
-        }}
-      >
-        {meetups.map(({ id, logo, title, date, venue, replay }) => (
+        {data.map(({ id, name, start_time, place }) => (
           <Flex
             key={id}
-            flexDirection="column"
-            width="100%"
-            maxWidth="30em"
-            color="darkblue"
-            mt={2}
-            mx="auto"
+            as="li"
+            bg="white"
+            p={2}
+            mb={2}
+            color="darkBlue"
+            alignItems={["start", "center"]}
+            flexDirection={["column", "row"]}
           >
-            <Image
-              src={logo}
-              alt={title}
-              width="12.5rem"
-              mx="auto"
-              mb={3}
-              display="block"
-            />
-
-            <Flex
-              bg="white"
-              p={2}
-              flexDirection="column"
-              flex={1}
+            <Box
+              as="time"
+              dateTime={start_time}
+              pr={2}
+              mr={2}
               sx={{
-                boxShadow: "main",
-                transition: "200ms transform",
-                "&:hover": {
-                  transform: "translateY(-0.5rem)",
-                },
+                textTransform: "uppercase",
+                borderRightWidth: "1px",
+                borderRightStyle: "solid",
+                borderRightColor: ["transparent", "darkBlue"],
               }}
             >
-              <Box flex={1}>
-                <Flex alignItems="baseline" fontWeight="bold" fontSize={2}>
-                  <Link href="/">{title}</Link>
-                  <Text ml="0.5rem">›</Text>
-                </Flex>
-                <Text my="0.25rem">{date}</Text>
-                <Text>{venue}</Text>
-              </Box>
+              {formatDate(start_time)}
+            </Box>
+
+            <Box flex={1} pt={[1, 0]} pb={[2, 0]}>
+              <Text
+                as="h4"
+                fontSize={[2, 3]}
+                fontWeight="heading"
+                sx={{ textTransform: "uppercase" }}
+              >
+                {name}
+              </Text>
+              <Text>{`${place.name}, ${place.location &&
+                place.location.city}`}</Text>
+            </Box>
 
-              <Box mt={2}>
-                {!replay ? (
-                  <>
-                    <Button variant="outline" mr={1}>
-                      △ Sponsor
-                    </Button>
-                    <Button variant="outline" mt="1">
-                      ▩ SPEAK
-                    </Button>
-                  </>
-                ) : (
-                  <Button variant="outline">◉ Watch</Button>
-                )}
-              </Box>
-            </Flex>
+            <Button
```

Comment:
Can you add `mr="6px"` and `mb="6px"` to account for the box shadow, so the padding on the container looks right?